Apps/WebClient/src/Server/Controllers/RobotsController.cs

```diff
@@ -58,5 +60,20 @@ public ActionResult Robots()
             };
             return result;
         }
+
+        [SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Team decision")]
+        private static string? ReadRobotsFile(string robotsFilePath)
+        {
+            string? defaultRobotsAssetContent = AssetReader.Read("HealthGateway.WebClient.Server.Assets.Robots.txt");
```

Comment:
Nit: this operation is performed even on paths where it isn't needed.  This could be avoided, e.g. by introducing a local function that reads this asset.

Reply:
Changed.